undertow/src/main/java/org/wildfly/extension/undertow/AccessLogDefinition.java

```diff
@@ -56,7 +56,7 @@
             .setDefaultValue(new ModelNode("default"))
             .build();
     protected static final SimpleAttributeDefinition PREFIX = new SimpleAttributeDefinitionBuilder(Constants.PREFIX, ModelType.STRING, true)
-            .setDefaultValue(new ModelNode("access_log"))
+            .setDefaultValue(new ModelNode("access_log."))
```

Comment:
As suffix is by default `.log` should be the suffix default also changed? Either to `log` or to the same value as was used in Web, where the suffix is counted from current date?